internal/slack/slack_test.go

```diff
@@ -36,21 +36,23 @@ func TestNewMutableClient_mapping(t *testing.T) {
 		}
 	)
 
-	slackClient := intslack.MockSlackClient{}
-	slackClient.Test(t)
-	slackClient.On("PostMessage", slackUser.ID, mock.Anything, mock.Anything).Return("", "", nil)
-	// return error when looking up non-corp email
-	slackClient.On("GetUserByEmailContext", mock.Anything, email).Return(&slack.User{}, errors.New("users_not_found"))
-	// return slack user on anything else
-	slackClient.On("GetUserByEmailContext", mock.Anything, mock.Anything).Return(slackUser, nil)
+	t.Run("domain verification", func(t *testing.T) {
+		slackClient := intslack.MockSlackClient{}
+		slackClient.Test(t)
+		slackClient.On("PostMessage", slackUser.ID, mock.Anything, mock.Anything).Return("", "", nil)
```

Comment:
You can make an assertion on the payload of the posted message here I think.

Reply:
I am not sure how to do that :) I am interested in `slack.MsgOptionAttachments` but these are mocked so no actually logic concerning them is not performed.

Reply:
Ah I see. The `slack.MsgOption` option pattern makes it hard to assert on what is provided. 👍 No worries.